src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+            </FieldGroup>
+            {formSubmitted ? (
+              <div className="dp-wrap-confirmation" style={containerMessageSucces}>
+                <div className="dp-msj-confirmation bounceIn" style={flextStartSyle}>
+                  <p>{_('checkoutProcessForm.success_msg')}</p>
+                </div>
+              </div>
+            ) : null}
+            <div className="dp-footer-form" style={flextStartSyle}>
+              <div className="dp-action">
+                <SubmitButton className="dp-button button-medium primary-green">
+                  {_('checkoutProcessForm.save_continue')}
+                </SubmitButton>
+              </div>
+            </div>
+          </fieldset>
+        </FormWithCaptcha>
+      ) : null}
+    </div>
+  );
+};
+
+const CheckoutProcess = ({
+  location,
+  dependencies: { planService, dopplerUserApiClient, appSessionRef },
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const { pathType, planType } = useRouteMatch().params;
+
+  const [steps, setSteps] = useState({
+    contactInformation: true,
+    billingInformation: false,
+    paymentInformation: false,
+  });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep();
+    }
+
+    return result;
+  };
+
+  const setActiveStep = (target) => {
+    switch (target) {
+      case 'contact-information':
+        setSteps({
+          contactInformation: true,
+          billingInformation: false,
+          paymentInformation: false,
+        });
+        break;
+      case 'billing-information':
+        setSteps({
+          contactInformation: false,
+          billingInformation: true,
+          paymentInformation: false,
+        });
+        break;
+      case 'payment-information':
+        setSteps({
+          contactInformation: false,
+          billingInformation: false,
+          paymentInformation: true,
+        });
+        break;
+    }
+  };
+
+  const [state, setState] = useState({ loading: true });
+
+  useEffect(() => {
+    const mapCountry = (country) => {
+      return {
+        key: country.id,
+        value: country.name,
+      };
+    };
+
+    const fetchData = async () => {
+      const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+      const countriesResult = await dopplerUserApiClient.getCountriesData();
+
+      const countries = countriesResult.success ? countriesResult.value.map(mapCountry) : [];
+      countries.splice(0, 0, { key: '', value: _('checkoutProcessForm.empty_option_select') });
+
+      setState({
+        contactInformation: contactInformationResult.success
+          ? contactInformationResult.value
+          : null,
```

Comment:
I think that if the result is not success you should explicitly represents an error

```suggestion
        contactInformation: contactInformationResult.success
          ? contactInformationResult.value
          : null,
        isError: !contactInformationResult.success,
```